setup/react/public/mockServiceWorker.js

```diff
@@ -8,121 +8,121 @@
  * - Please do NOT serve this file on production.
```

Comment:
From the comments and lint disable above, it feels like this file should not be modified. I dont know why thought 🤔.